.github/workflows/e2e.yml

```diff
@@ -0,0 +1,61 @@
+name: E2E
+
+on: [push, pull_request]
```

Comment:
@aloisklink Se my comment regarding when to trigger

Reply:
I've deleted the commit adding Applitools to CI, and will create a new PR that adds a `e2e-applitool.yml` CI action.

The `e2e.yml` test now doesn't use Applitools, so it's fine to have it run on every push/PR, since GitHub Actions is currently completely free for open-source projects.